website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -0,0 +1,146 @@
+---
+description: >-
+  Appsmith ships with third-party services & configurations that improve the app
+  building experience. All services & configurations are entirely optional.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, simply open the folder of your installation and edit the **`stacks/configuration/docker.env`** file with the environment variables for the service.
+
+Remove # before the variables to ensure they're not commented
+
+```bash
+// Example variable configuration format
+APPSMITH_GOOGLE_MAPS_API_KEY=YOUR_API_KEY
+```
+
+After making any changes, **remember to restart the docker containers** for the changes to take effect.
+
+```bash
+// To restart Appsmith using docker
+docker restart appsmith
+
+// To restart Appsmith using docker compose
+docker-compose restart appsmith
+```
+
+## Configure Kubernetes installations
+
+To configure a `k8s` installation, simply open the folder of your installation and edit the `config-template/appsmith-configmap.yaml` file.
+
+Remove # before the variables to ensure they're not commented
+
+```
+// Example variable configuration format
+APPSMITH_GOOGLE_MAPS_API_KEY: "YOUR_API_KEY"
+```
+
+After making any changes, **remember to restart the pods** for the changes to take effect
+
+```
+// commands to restart k8s pods
+kubectl apply -f appsmith-configmap.yaml
+kubectl scale deployment appsmith-internal-server --replicas=0
+kubectl scale deployment appsmith-internal-server --replicas=1
+```
+
+## Configure Helm installations
+
+To configure environment variables using helm, follow these steps:
+
+<Tabs groupId="appsmithEditions">
+  <TabItem label="Community Edition" value="communityEdition"> 
+
+  You may choose to configure environment variables: through commands or by using a `values.yaml` file.
+
+  * When using command, specify each parameter using `--set key=value[,key=value]` argument to helm upgrade.
+
+```bash
+  helm upgrade --set applicationConfig.<ENV_KEY>=<ENV_VALUE> [ RELEASE ] [ CHART ]  
+```
+  For example, the below command sets the `APPSMITH_SIGNUP_DISABLED` variable to `true` and configures it to disable signup button on Appsmith login screen. 
```

Comment:
Ensure your code blocks and content line up to the bullet point's content, so they all show up aligned in the rendered website. In this case, we currently have this:

![Screenshot 2022-12-29 at 10 14 13 AM](https://user-images.githubusercontent.com/120119/209904722-bf50f4ef-8924-4d3b-9d08-e9e48b04779d.png)


If we align to the content of the bullet point, which is `When using...`:
![Screenshot 2022-12-29 at 10 15 40 AM](https://user-images.githubusercontent.com/120119/209904581-ceab35a6-faf4-4412-9897-aaca0c8e6ecd.png)

Then we get this:

![Screenshot 2022-12-29 at 10 16 51 AM](https://user-images.githubusercontent.com/120119/209904730-6a530604-7125-4c37-bcc8-2825e90043a6.png)

See the difference? Can we fix this for all the other code blocks under bullet points as well please?

Reply:
Got it.